### PR TITLE
[POC] - Adding multilanguage support

### PR DESCRIPTION
**Pull request by rodrigo-cervone**
Implements a module to translate the UI

### CODE REVIEW

File: src/pages/user-details/user-details.ts [hunk 1]
```diff
@@ -39,4 +50,14 @@ export class UserDetailsPage {
   goToRepos(login: string) {
     this.navCtrl.push(ReposPage, {login});
   }
+
+  changeLanguage() {
```
Comment (jhcruz) [inline src/pages/user-details/user-details.ts]: Documentation

File: src/pages/user-details/user-details.ts [hunk 2]
```diff
@@ -39,4 +50,14 @@ export class UserDetailsPage {
...
+    this.i18nService.use(this.isSpanish ? LANG_ES_NAME : LANG_EN_NAME);
+  }
+
+  onToggleChange() {
```
Comment (jhcruz) [inline src/pages/user-details/user-details.ts]: documentation

File: src/i18n/i18n.service.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { LANG_ES_TRANS } from './lang/dict.es';
+import { LANG_EN_TRANS } from './lang/dict.en';
+
+export const LANG_EN_NAME = 'en';
+export const LANG_ES_NAME = 'es';
```
Comment (jhcruz) [inline src/i18n/i18n.service.ts]: Is not better use it this const on an array?

File: src/i18n/lang/dict.es.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const LANG_ES_TRANS = {
```
Comment (jhcruz) [inline src/i18n/lang/dict.es.ts:1]: the solution could to apply a complex lines? like "Welcome {name}", where "name" is a variable

File: src/app/app.module.ts [hunk 5]
```diff
@@ -9,6 +9,12 @@ import { UsersPage } from '../pages/users/users';
 import { ReposPage } from '../pages/repos/repos';
 import { UserDetailsPage } from '../pages/user-details/user-details';
 import { UserLoginPage } from '../pages/user-login/user-login';
+import { I18nModule } from '../i18n/i18n.module';
```
Comment (jhcruz) [inline src/app/app.module.ts:12]: this module is from Angular?
import { TRANSLATIONS, TRANSLATIONS_FORMAT, LOCALE_ID } from '@angular/core';
https://angular.io/docs/ts/latest/cookbook/i18n.html

**Review by jhcruz — CHANGES_REQUESTED** [review]

File: src/pages/user-details/user-details.ts (hunk 1 above)
Comment (rodrigo-cervone) [reply src/pages/user-details/user-details.ts]: sure

File: src/pages/user-details/user-details.ts (hunk 2 above)
Comment (rodrigo-cervone) [reply src/pages/user-details/user-details.ts]: sure *2

File: src/i18n/i18n.service.ts (hunk 3 above)
Comment (rodrigo-cervone) [reply src/i18n/i18n.service.ts]: could be, I'll change it

File: src/i18n/lang/dict.es.ts (hunk 4 above)
Comment (rodrigo-cervone) [reply src/i18n/lang/dict.es.ts:1]: Didn't try that. I'll change it to see what happens.

File: src/app/app.module.ts (hunk 5 above)
Comment (rodrigo-cervone) [reply src/app/app.module.ts:12]: mmm... no... is a custom implementation. I didn't saw that module. Good catch.

**Comment by urieljbm** [comment]
All looks fine.

**Pull request merged**